sdk/storage/azure-storage-file-share/azure/storage/fileshare/_directory_client.py

```diff
@@ -348,8 +348,9 @@ def create_directory(self, **kwargs):
                 This parameter was introduced in API version '2021-06-08'.
 
         :paramtype file_change_time: str or ~datetime.datetime
-        :keyword dict[str, str] metadata:
-            Name-value pairs associated with the directory as metadata.
+        :keyword metadata:
+            Name-value pairs associated with the share as metadata.
```

Comment:
nit: you changed the wording of just this description. `directory` -> `share`. I think `directory` is correct since this is for `create_directory`

Reply:
Good catch!